src/rabbit-event-bus/amqp-connector.ts

```diff
@@ -75,9 +70,14 @@ export default class AMQPConnector {
                     logger.fatal(`Can't create subscriber queues for: ${this.serviceName} using event: ${eventType}`);
                 });
         } else {
+            logger.warn("No CONTAINER, can't subscribe, trying again soon!");
```

Comment:
:grin: Can we have a better message? `Could not connect to <params>, trying again in <n> seconds`

Reply:
Url will contain username + password in the connection string... I guess we should use serviceName. @diversemix any thoughts?

Reply:
I've used the previous message https://github.com/libero/event-bus/blob/master/src/rabbit-event-bus/amqp-connector.ts#L80 and adjusted slightly.